src/index.html

```diff
@@ -2,16 +2,181 @@
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+            class="input"
+            value="12"
+          >
+        </label>
+        <label class="formfield last">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            class="input"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="formfield">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            class="input"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="formfield last">
+          Password:
+          <input
+            type="password"
+            name="password"
+            class="input"
+            maxlength="8"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <label class="formfield">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            class="input"
+            value="yes"
+            id="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+            type="radio"
+            name="cats"
+            class="input"
+            value="no"
+            id="no"
+          >
+          <label for="no">No</label>
+        </label>
+        <label class="formfield">
+          What is your favourite color?
+          <input
+            type="color"
+            name="color"
+            class="input"
+          >
+        </label>
+        <label class="formfield">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            class="input"
+          >
+        </label>
+        <label class="formfield">
+          What are your favorite brand of cars?
+          <select
+            name="cars"
+            class="input"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="tesla">Tesla</option>
+          </select>
+        </label>
+        <label class="formfield last">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            class="input"
+            min="1"
+            max="5"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Additional info:</legend>
+        <label class="formfield">
+          Comments:
+          <textarea
+            class="input"
+            name="comments"
+            cols="20"
+            rows="2"
+            autocomplete="off"
+            maxlength="500"
+          >
+          </textarea>
+        </label>
```

Comment:
```suggestion
            maxlength="500"
        ></textarea>
        </label>
```

https://user-images.githubusercontent.com/102413088/193272330-7e8bf972-a95c-4762-a4bc-a3edf5fa12da.mov